### DISCUSSION TITLE
How to validate schema with references schema stored in a database

### DISCUSSION THREAD

**Discussion by Gustutu** [python-jsonschema/jsonschema#1077] [Q&A] [answered]
Hi, i have a schema wich will refers to schema stored in a database as JSON object, what modification/code i should write to make it work.
I am just asking the global idea to go in the right direction.

**Comment by Julian** [comment, marked as answer]
This has changed (and hopefully is clearly easier) in v4.18, which is currently in prerelease.

Care to have a look at [the referencing docs](https://python-jsonschema.readthedocs.io/en/latest/referencing/) and leave feedback if you still can't tell how to do this after reading it?

**Reply by Gustutu** [reply]
Awesome, i'll look at it soon. I'll let you know my success. Thanks.
